### PR TITLE
Ant Mission 2

### PR DESCRIPTION
**Pull request by Cheesy0314**
Separated out mission 2 from mission 1 (#15396)

Cleaned up lua code, pulled image from cnc-wiki and rescaled.

map.bin and most of map.yaml are from #15213

Altered some actors/factions to be more in line with original game.

### CODE REVIEW

**Comment by Cheesy0314** [comment]
The travis-cli is failing on something added from upstream/bleed, not sure how to fix that

**Comment by TheChosenEvilOne** [comment]
It is failing on your script

```
Checking for Lua syntax errors...
luac: mods/ra/maps/ant-02/ant-02.lua:143: unexpected symbol near ')'
make: *** [check-scripts] Error 1
The command "make check-scripts" exited with 2.
```

**Comment by Cheesy0314** [comment]
Ahh I see, I’m an idiot. Don’t know how I missed that.

Thank you

**Comment by Cheesy0314** [comment]
Update: Applied some of the same styling/rules/coding practice changes to this PR that were requested in Ant Mission 1's PR

**Comment by Cheesy0314** [comment]
Changed colors, Player names, and separated scripts into main (ant-02.lua) and ant-attack (ant-attack.lua)

**Comment by Smittytron** [comment]
- Gave this a play-through. My first impression is that the difficulty is dialed up to 11; on easy. Hard isn't winnable in it's current state.

- I say that, but I was able to 'win' on hard with an exploit. If you bum-rush your rangers at the start to the south anthill you can take it out. This also stopped the top hills from spawning ants, resulting in only four or so ants spawned the entire mission. I think this might be with how the starting triggers are set up, because when I beat the mission the first time on easy I killed the south anthill and the north kept spitting out ants just fine.

- Civilian structures show their health bars under fog and shroud.

**Comment by Cheesy0314** [comment]
Yeah I need to go back over this one now that It's changed so much

**Comment by pchote** [comment]
What is the review status here @abcdefg30, @Smittytron, @reaperrr?

**Comment by Cheesy0314** [comment]
@pchote Pretty confident this one needs work, have to test it again soon.

File: mods/ra/maps/ant-02/map.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ Players:
 		Color: F6D679
 		LockSpawn: True
 		LockTeam: True
-		Allies: GoodGuy, Special, Neutral
+		Allies: Special,
```
Comment (Mailaender) [inline mods/ra/maps/ant-02/map.yaml]: The last `,` looks bogus.

**Comment by pchote** [comment]
No progress here in 6 months, so moving to the Future milestone and closing to help tidy the PR queue.

**Pull request closed without merging**